cmd/gangway/handlers.go

```diff
@@ -30,6 +32,7 @@ import (
 
 const (
 	templatesBase = "templates"
+	clusterCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
This forces gangway to be run in-cluster. There could be scenarios where this is undesirable and/or a different CA may want to be used.  Can we have a config option with this as the default?